addons/languages/kurdish/pack/src/main/res/xml/kurdish_qwerty.xml

```diff
@@ -1,59 +1,40 @@
+        <Key android:codes="1608" android:popupKeyboard="@xml/popup_w" android:keyLabel="و" android:popupCharacters="۲"/>
+        <Key android:codes="1749" android:keyLabel="ﻪ" android:popupCharacters="۳"/>
+        <Key android:codes="1585" android:keyLabel="ر" ask:shiftedCodes="1685" android:popupKeyboard="@xml/popup_r"/>
         <Key android:codes="1578" android:keyLabel="ت" android:popupCharacters="٥"/>
-        <Key android:codes="1740" android:keyLabel="ی" ask:shiftedCodes="1742" ask:shiftedKeyLabel="ێ" android:popupKeyboard="@xml/popup_qwerty_y"/>
+        <Key android:codes="1740" android:keyLabel="ی" ask:shiftedCodes="1742" android:popupKeyboard="@xml/popup_y"/>
         <Key android:codes="1574" android:keyLabel="ئ" android:popupCharacters="٧"/>
-        <Key android:codes="1581" android:keyLabel="ح" ask:shiftedCodes="1593" ask:shiftedKeyLabel="ع" android:popupCharacters="ع٨"/>
-        <Key android:codes="1734" android:keyLabel="ۆ" android:popupCharacters="٩"/>
+        <Key android:codes="1581" android:keyLabel="ح" ask:shiftedCodes="1593" android:popupCharacters="ع٨"/>
+        <Key android:codes="1734" android:keyLabel="ۆ" android:popupCharacters="ۋۉۊؤ٩"/>
         <Key android:codes="1662" android:keyLabel="پ" android:popupCharacters="٠؛،" android:keyEdgeFlags="right"/>
     </Row>
 
     <Row>
-        <Key android:horizontalGap="5%p" android:codes="1575" android:keyLabel="ا" android:keyEdgeFlags="left"/>
+        <Key android:horizontalGap="5%p" android:codes="1575" android:keyLabel="ا" android:popupCharacters="آإأ" android:keyEdgeFlags="left"/>
         <Key android:codes="1587" android:keyLabel="س" ask:shiftedCodes="1588" ask:shiftedKeyLabel="ش" android:popupCharacters="ش" />
         <Key android:codes="1583" android:keyLabel="د" />
-        <Key android:codes="1601" android:keyLabel="ف" />
+        <Key android:codes="1601" android:keyLabel="ف" android:popupCharacters="﷽ﷲﷻ" />
         <Key android:codes="1711" android:keyLabel="گ" ask:shiftedCodes="1594" ask:shiftedKeyLabel="غ" android:popupCharacters="غ" />
-        <Key android:codes="1607" android:keyLabel="ھ" ask:shiftedKeyLabel="ھ" />
+        <Key android:codes="1607" android:keyLabel="ھ" android:popupCharacters="ﷴﷺ" />
         <Key android:codes="1688" android:keyLabel="ژ" />
-        <Key android:codes="1705" android:keyLabel="ک" />
+        <Key android:codes="1705" android:keyLabel="ک" android:popupCharacters="ڪك" />
         <Key android:codes="1604" android:keyLabel="ل" ask:shiftedCodes="1717" ask:shiftedKeyLabel="ڵ" android:popupCharacters="ڵ" android:keyEdgeFlags="right"/>
     </Row>
 
     <Row>
-        <Key android:codes="-1" android:keyWidth="15%p" android:isModifier="true" android:isSticky="true" android:keyEdgeFlags="left"/>
+        <Key android:keyWidth="13.999998%p" android:codes="1600" android:keyEdgeFlags="left" android:isRepeatable="true" android:keyIcon="@drawable/drezh" ask:isFunctional="true" />
```

Comment:
this is interesting... why do you need this drawable?

Reply:
That drawbale (drezh) is to know that button is for  stretch characters(Arabic Tatweel) because the unicode=1600 doen not have a character preview. 

look at this you know what i means :) 

 https://www.fileformat.info/info/unicode/char/0640/index.htm